src/main/java/com/wfector/command/CommandRunner.java

```diff
@@ -134,7 +134,7 @@ public boolean onCommand(final CommandSender sender, Command cmd, String label,
                             if (userId == null) {
                                 OfflinePlayer target = plugin.getServer().getPlayer(args[1]);
                                 if (target == null) {
-                                    target = plugin.getServer().getOfflinePlayer(args[1]);
+                                    target = plugin.getServer().getOfflinePlayer(NameManager.getUUID(args[1]));
```

Comment:
is it possible to di this method instead of the deprecated `getOfflinePlayer(args[1])` method?

Reply:
There is not really a need to get the OfflinePlayer here as it was only used to get the player's UUID which can be replaced with the NameManager method call.